app/fissile.go

```diff
@@ -297,6 +297,38 @@ func (f *Fissile) collectProperties() map[string]map[string]map[string]interface
 	return result
 }
 
+// propertyDefaults is a double map
+//
+//	(property.name -> (default.string -> [*job...])
+//
+// which maps a property (name) to all its default values
+// (stringified) and them in turn to the array of jobs where this
+// default occurs.
+type propertyDefaults map[string]map[string][]*model.Job
+
+func (f *Fissile) collectPropertyDefaults() propertyDefaults {
+	result := make(propertyDefaults)
+
+	for _, release := range f.releases {
+		for _, job := range release.Jobs {
+			for _, property := range job.Properties {
+				defaultAsString := fmt.Sprintf("%v", property.Default)
+
+				if _, ok := result[property.Name]; !ok {
+					result[property.Name] = make(map[string][]*model.Job)
+				}
+				if _, ok := result[property.Name][defaultAsString]; !ok {
+					result[property.Name][defaultAsString] = make([]*model.Job, 1)
```

Comment:
`(0,1)` is definitely more correct (length and capacity).
Well, can disable and see if the code still works.
... Reading up on the index expressions I see `if the map is nil or does not contain such an entry, a[x] is the zero value for the value type of M`, so you are correct, this should work if `append` also accepts `nil` in the 1st argument. Testing ... Works ... Will remove the entire block.